diff --git a/app/config/email.php b/app/config/email.php
index 36e878b..b95fe85 100644
--- a/app/config/email.php
+++ b/app/config/email.php
@@ -18,7 +18,7 @@ return array(
     'ask.ProfileImagePath' => '/Areas/Admin/Content/images/btn_learn_lawyer.png',
     'ask.reply.subject'    =>'Answer to Your Reply - Ask a Lawyer - Lawyers.com',
     'ask.reply.fromAddress' => 'NoReply@lawyers.com',
-    'toEmailAddress' => '',
+    'toEmailAddress' => 'qatester@martindale.com;nagendrakumar.s@martindale.com;valliammai.subramanian@martindale.com',
      //Delete - Question Email Parameters
     'ask.deleteQuestion.fromAddress'       => 'NoReply@lawyers.com',
     'ask.deleteQuestion.Subject' => 'Your Question Has Been Removed - Lawyers.com',
